system/lib/pthread/library_pthread.c

```diff
@@ -978,3 +978,31 @@ void __do_cleanup_push(struct __ptcb *cb) {
 void __do_cleanup_pop(struct __ptcb *cb) {
   __pthread_self()->cancelbuf = cb->__next;
 }
+
+extern int __pthread_create_js(pthread_t *thread, const pthread_attr_t *attr, void *(*start_routine) (void *), void *arg);
+int __pthread_create(pthread_t *thread, const pthread_attr_t *attr, void *(*start_routine) (void *), void *arg) {
```

Comment:
Can't you make `__pthread_create` itself a weak alias to `__pthread_create_js`?

Reply:
No, I think the target of the alias must be a defined function.. and js functions are always undefined.